### PR TITLE
fix(RBAC): RHICOMPL-2875: Revert RHICOMPL-2874

### PR DESCRIPTION
**Pull request by romanblanco**
Reimplement RBAC loader

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by skateman** [comment]
Bumping the lockfile like this is possibly dangerous ...

File: config/initializers/rbac.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RBACApiClient.configure do |config|
+  config.host = Settings.rbac_url
+  config.username = Settings.platform_basic_auth_username
```
Comment (romanblanco) [inline config/initializers/rbac.rb:3]: don't send if `nil`.

File: app/services/rbac.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'insights-rbac-api-client'
+
+# This service is meant to handle calls to the RBAC API
+class Rbac
+  API_CLIENT = RBACApiClient::AccessApi.new
+  COMPLIANCE_FULL_ACCESS = 'compliance:*:*'
+  APPS = 'compliance,inventory'
+  OPTS = { limit: 10, offset: 0 }.freeze
```
Comment (skateman) [inline app/services/rbac.rb:10]: I guess this one :see_no_evil:

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
525c3af7-b675-49d4-a751-8a7a90e74c66
5a1de148-13e4-406d-9991-06824117613d

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
48290a2b-68d8-4682-a10c-ce6b3860d812
c9b48b07-3b4d-459d-bc69-a6d6820c1fa0

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
11965db8-b6cd-4921-a35d-46c54e364635
e4fcbc74-d6ba-4159-9300-93fed50744a3

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
2fe76f3a-196a-4a87-a6e9-769c0d6629b0
70caf170-4999-4b2a-9d02-e1be16bd6e07

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
2c950e87-ff3e-4309-9608-348438e1d0b9
cef937e2-1a3e-4b16-a4f4-c9da8bb30e74

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
33526283-c449-443b-b188-aab16f502d39
958a0863-19d5-49d9-b912-08e3d07c52e1

**Review by skateman — APPROVED** [review]
The Seal of Approval
Nice job :+1:, please hold off the merge until the last card in the Next release is verified

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
34c9ed3a-a81c-49ea-b8f3-4a118dbbfc9b
dc74cc23-1f98-4097-b110-02b3c074c43e

**Pull request merged**